xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4553,7 +4553,7 @@ void CVideoPlayer::AddSubtitle(const std::string& strSubPath)
 bool CVideoPlayer::IsCaching() const
 {
   CSingleLock lock(m_StateSection);
-  return m_State.caching;
+  return !IsInMenuInternal() && m_State.caching;
```

Comment:
IsInMenuInternal can only be called from player thread. Use m_State.isInMenu here